design/ceph/object/ceph-sse-s3.md

```diff
@@ -26,11 +26,27 @@ security:
       VAULT_ADDR: https://vault.default.svc.cluster.local:8200
       VAULT_SECRET_ENGINE: transit
       VAULT_AUTH_METHOD: token
+      ENCRYPT_ALL_OBC: true
     # name of the k8s secret containing the kms authentication token
     tokenSecretName: rook-vault-token
 ```
 The values for this configuration will be available in [Security field](Documentation/CRDs/Object-Storage/ceph-object-store-crd.md#security-settings) of `CephObjectStoreSpec`, so depending on the above option RGW can be configured with `SSE-KMS` and `SSE-S3` options. These two options can be configured independently and they both are mutually exclusive in Rook and Ceph level.
 
+The `ENCRYPT_ALL_OBC` will set [PutBucketEncryption](https://docs.aws.amazon.com/AmazonS3/latest/userguide/default-bucket-encryption.html) on all the bucket provisioned via obc by the Rook. So the all objects in that obc will encyrpt using `SSE-S3`. This can done manually via
+s3 api using following:
+```
+aws s3api put-bucket-encryption --no-verify-ssl --endpoint-url http://$BUCKET_HOST:$BUCKET_PORT --bucket $BUCKET_NAME  --server-side-encryption-configuration '{
```

Comment:
As discussed, is there some way RGW can enforce this for all buckets? Rook can only set this on buckets created for OBCs, but not any arbitrary bucket if created separately from OBCs.